tests/jest/aggregate-messages.test.js

```diff
@@ -6,7 +6,7 @@ global.console = { warn: jest.fn() };
 
 const translationDirectories = [
   path.resolve(__dirname, 'fixtures', 'translations'),
-  path.resolve(__dirname, 'fixtures', 'node_modules', 'fixtures1', 'translations'),
+  path.resolve(__dirname, 'fixtures', 'ned_modules', 'fixtures1', 'translations'),
```

Comment:
wat?

Reply:
Let's kick it off with a nice White Wine Spriter then